modules/n1ql/pages/n1ql-language-reference/prepare.adoc

```diff
@@ -35,17 +35,14 @@ To execute the following statement, user must have the _Query Select_ privilege
 
 [source,sqlpp]
 ----
-PREPARE SELECT * FROM airport
-WHERE city = (SELECT RAW city FROM landmark)
+include::example$n1ql-language-reference/prepare-rbac-select.n1ql[]
```

Comment:
This directory is for DDL examples, can you put this example in the `utility` directory?